src/state_manager/utils.rs

```diff
@@ -102,7 +102,12 @@ where
 
         let out = sectors
             .into_iter()
-            .map(|s_info| SectorInfo::new(*spt, s_info.sector_number, s_info.sealed_cid))
+            .map(|s_info| ExtendedSectorInfo {
```

Comment:
How about implementing `From<SectorInfo> for ExtendedSectorInfo`

Reply:
the `s_info` is not a `SectorInfo` but a `SectorOnChainInfo`. I'd leave it as is, given the conversion would be used only in a single place.